recipe/meta.yaml

```
@@ -22,40 +22,57 @@ build:
     - fitsinfo = astropy.io.fits.scripts.fitsinfo:main
     - samp_hub = astropy.samp.hub_script:hub_script
     - showtable = astropy.table.scripts.showtable:main
+    - showtable-astropy = astropy.table.scripts.showtable:main
     - volint = astropy.io.votable.volint:main
     - wcslint = astropy.wcs.wcslint:main
+  skip: true  # [py<311]
 
 requirements:
   build:
     - {{ compiler('c') }}
     - pkg-config
+    - patch  # [not win]
+    - m2-patch  # [win]
```

```suggestion
    - msys2-patch  # [win]
```
We should switch to `msys2-patch`.